h5c/vic/src/vic-webapp/src/app/shared/components/vch-network/vch-network.component.html

```diff
@@ -0,0 +1,823 @@
+                     placeholder="Port"
+                     type="text">
+              <span class="tooltip-content" *ngIf="form.get('httpProxyPort').hasError('maxlength')">
+                {{ i18n.translate('vch.storage.network.portDigits') }}
+              </span>
+              <span class="tooltip-content" *ngIf="form.get('httpProxyPort').hasError('pattern')">
+                {{ i18n.translate('vch.storage.network.portNumeric') }}
+              </span>
+            </label>
+          </div>
+          <!-- To set previous columns widths, we need to set space for 2 dynamic width columns -->
+          <div class="col-xs"></div>
+          <div class="col-xs"></div>
+        </div>
+
+        <!-- HTTPS Proxy -->
+        <div class="form-group row">
+          <div class="col-xs-3">
+            <label for="https-proxy-selector">{{ i18n.translate('vch.storage.network.httpsProxy') }}</label>
+          </div>
+          <div class="col-xs-3">
+            <input id="https-proxy-selector"
+                   class="form-control"
+                   placeholder="IP or FQDN"
+                   type="text"
+                   formControlName="httpsProxy">
+            <small class="text-muted text-nowrap">
+              {{ i18n.translate('vch.storage.network.note') }}
+            </small>
+          </div>
+          <span class="mx-0 px-0">:</span>
+          <div class="col-xs-1">
+            <label for="https-proxy-port-selector"
+                   aria-haspopup="true"
+                   role="tooltip"
+                   class="tooltip tooltip-validation tooltip-md tooltip-top-left"
+                   [class.invalid]="form.get('httpsProxyPort').invalid && (form.get('httpsProxyPort').dirty || form.get('httpsProxyPort').touched)">
+              <input id="https-proxy-port-selector"
+                     placeholder="Port"
+                     type="text"
+                     formControlName="httpsProxyPort">
+              <span class="tooltip-content" *ngIf="form.get('httpsProxyPort').hasError('maxlength')">
+                {{ i18n.translate('vch.storage.network.portDigits') }}
+              </span>
+              <span class="tooltip-content" *ngIf="form.get('httpsProxyPort').hasError('pattern')">
+                {{ i18n.translate('vch.storage.network.portNumeric') }}
+              </span>
+            </label>
+          </div>
+          <!-- To set previous columns widths, we need to set space for 2 dynamic width columns -->
+          <div class="col-xs"></div>
+          <div class="col-xs"></div>
+        </div>
+      </section>
+
+      <!-- Optional Container Networks -->
+      <section class="form-block" *ngIf="inAdvancedMode">
+        <label>{{ i18n.translate('vch.storage.network.optionalContainerNetworks') }}</label>
+        <div class="mb-2"
+             formArrayName="containerNetworks"
+             *ngFor="let containerNetwork of form.get('containerNetworks').controls; let i=index">
```

Comment:
why not just reuse 'index', instead of re-assigning to 'i'?

Reply:
Most of the code of the template was copied from the original one, so I didn't change it:
https://github.com/vmware/vic-ui/blob/e8e3267cacb84b3e60770c43dbe0dc0dfc04341c/h5c/vic/src/vic-webapp/src/app/create-vch-wizard/networks/networks.html#L611

as a side note, AFAIK you have to declare the variable to get access to the index on an *ngFor . There is no way to get access to the index if you don't assign it to a variable.